packages/bits/spec/virtual-camera/PercyLens.ts

```diff
@@ -13,7 +13,7 @@ export class PercyLens implements ILens {
     public async takeSnapshot(label: string): Promise<void> {
         this.checkSettings();
```

Comment:
It might be slightly nicer to have an observable on the camera that the lens can subscribe to for settings updates. That way you would have to check the settings every time you take a snapshot. Not a requirement, just a suggestion.